packages/engine/lib/error/src/report.rs

```diff
@@ -196,28 +199,28 @@ impl<C> Report<C> {
     }
 
     /// Creates an iterator over the [`Frame`] stack requesting references of type `T`.
-    pub const fn request_ref<T: ?Sized + 'static>(&self) -> RequestRef<'_, T> {
+    pub const fn request_ref<R: ?Sized + 'static>(&self) -> RequestRef<'_, R> {
         RequestRef::new(self)
     }
 
     /// Creates an iterator over the [`Frame`] stack requesting values of type `T`.
-    pub const fn request_value<T: 'static>(&self) -> RequestValue<'_, T> {
+    pub const fn request_value<R: 'static>(&self) -> RequestValue<'_, R> {
         RequestValue::new(self)
     }
 
     /// Returns if `T` is the type held by any frame inside of the report.
```

Comment:
I know we've agreed to do a clean-up of generic type params after this but can we keep them consistent until then (it's T but C in the code)

Reply:
Tried to be more consistent. Also made `Report::generalize` hidden in docs: 93976d12ba6d95de80370bbf3817cff9d050b538